mypy/messages.py

```diff
@@ -253,14 +253,17 @@ def format_simple(self, typ: Type, verbosity: int = 0) -> str:
                 base_str = itype.type.fullname()
             else:
                 base_str = itype.type.name()
-            if itype.args == []:
+            if itype.args == [] or len(itype.args) == 1 and type(itype.args[0]) == AnyType:
```

Comment:
My other changes caused a large number of `[Any]` to be added to error messages. I don't think error messages are improved by that, and even if they are, it would require changing a lot of updates to test cases' expected output.